javalin-testtools/src/main/java/io/javalin/testtools/DefaultSseTestHandler.kt

```diff
@@ -0,0 +1,10 @@
+package io.javalin.testtools
+
+abstract class DefaultSseTestHandler : SseTestHandler {
+
+    override fun onMessage(sseEvent: SseEvent?) {
+    }
+
+    override fun onFailure(sseFailure: SseFailure) {
+    }
+}
```

Comment:
I think you can remove this whole class if you use the the `default` keyword for the methods in `SseTestHandler`